tickets/src/main/java/no/unit/nva/publication/ticket/UpdateTicketOwnershipRequest.java

```diff
@@ -3,10 +3,16 @@
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import java.net.URI;
+import java.util.Optional;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 @JsonTypeName(UpdateTicketOwnershipRequest.TYPE)
 public record UpdateTicketOwnershipRequest(URI ownerAffiliation, URI responsibilityArea) implements TicketRequest {
 
     public static final String TYPE = "UpdateTicketOwnershipRequest";
+
+    @Override
+    public  URI responsibilityArea() {
+        return Optional.ofNullable(responsibilityArea).orElse(ownerAffiliation);
+    }
```

Comment:
Fixing in the same pull request this thing. When transferring ticket ownership, and use does not provide `responsibilityArea`, use `ownerAffiliation` as `responsibilityArea`